MainMenu.php

```diff
@@ -159,22 +159,22 @@
         <p>Thanks to the efforts of OotTheMonk, Brubraz, and ninin, many remaining Set 3 cards have now been implemented in Karabast.</p>
         <p>Remaining cards:</p>
         <ul class="karabast-unimplemented-list">
```

Comment:
it might be best to do something like 
```css
.karabast-unimplemented-list {
  display: block;
  margin-bottom: 1rem;
  list-style-type: disc;
  list-style-position: inside;
}

.karabast-unimplemented-list li {
  display: list-item;
}
```

Reply:
but this would require renaming the css file again..

Reply:
Nice idea! Thanks!